app/assets/stylesheets/flash.scss

```diff
@@ -0,0 +1,21 @@
+.flash-error {
+  color: #ff0000;
```

Comment:
Color literals like `#ff0000` should only be used in variable declarations; they should be referred to via variable everywhere else.